src/graphics-private.h

```diff
@@ -105,8 +106,12 @@ typedef struct _Graphics {
 	cairo_t			*ct;
 	GpMatrix		*copy_of_ctm;
 	cairo_matrix_t		previous_matrix;
+#if HAS_X11
+#ifdef CAIRO_HAS_XLIB_SURFACE
 	Display			*display;
 	Drawable		drawable;
+#endif
+#endif
```

Comment:
this struct isn't passed between managed<->native right? otherwise we might get issues if it changes size/layout.